src/tactics.ml

```diff
@@ -113,3 +113,41 @@ let handle_tactic : sig_state -> Proof.t -> p_tactic -> Proof.t =
       handle_refine (Rewrite.reflexivity tac.pos ps)
   | P_tac_sym           ->
       handle_refine (Rewrite.symmetry tac.pos ps)
+  | P_tac_why3(s)    ->
+      (* get the goal to prove *)
+      let (hypothesis, trm) = Proof.Goal.get_type g in
+      let trm = unfold trm in
+      (* get the default or the indicated name of the prover. *)
+      let prover_name =
+        match s with
+        | None          -> Timed.(!Why3prover.current_prover)
+        | Some(name)    -> name
+      in
+      (* translate from lambdapi to why3 terms. *)
+      let (l_prop, hyps, why3term) =
+          Why3prop.translate tac.pos ps.proof_builtins (hypothesis, trm) in
+      (* create a new task that contains symbols, axioms and the goal. *)
+      let tsk = Why3task.create l_prop hyps why3term in
+      (* call the prover named [prover_name] and get the result. *)
+      let prover_result = Why3prover.call  tac.pos prover_name tsk in
```

Comment:
double space after call?

Reply:
Fixed in 9d60b154731ebf77c5a6788b9ac65c938ef04521